stack.yaml

```diff
@@ -54,7 +54,7 @@ extra-deps:
 # Not strictly a dependency at present, but building it here to get
 # access to the cardano-tx cli.
```

Comment:
This comment no longer applies, so should be removed.

Reply:
cleaned in https://github.com/input-output-hk/cardano-wallet/pull/2043/commits/ae1f415d5a873eb3a50e501452e698ef6b6c71b9